locale/en/Pacifist.cfg

```diff
@@ -33,11 +33,11 @@ combat=Equipment
 enemies=Units
 
 [map-gen-preset-description]
-default=You have to turn off biters yourself - unless you like to suffer.
+default=Normal settings. The recommended way to play Factorio. There will be no biters due to Pacifist.
```

Comment:
Better wording, thanks :-)

Reply:
Before this PR, the biters did show up in the map preview, although they aren't there once the game starts.